cms/djangoapps/contentstore/rest_api/v1/serializers/course_waffle_flags.py

```diff
@@ -28,6 +28,7 @@ class CourseWaffleFlagsSerializer(serializers.Serializer):
     use_new_textbooks_page = serializers.SerializerMethodField()
     use_new_group_configurations_page = serializers.SerializerMethodField()
     enable_course_optimizer = serializers.SerializerMethodField()
+    react_markdown_editor = serializers.SerializerMethodField()
```

Comment:
```suggestion
    use_react_markdown_editor = serializers.SerializerMethodField()
```
to clarify that this is a boolean, and for consistency with the other flags in this serializer